src/shell/shell_cmds.cpp

```diff
@@ -938,7 +1140,7 @@ void DOS_Shell::CMD_COPY(char * args) {
 			return;
 		}
 
-		Bit16u sourceHandle,targetHandle;
+		Bit16u sourceHandle,targetHandle=65535;
```

Comment:
@dreamer The PVS analyzer seemed to show that the "targetHandle" variable may be uninitialized, so I tried to fix this warning by assigning it to an initial value. Perhaps not needed.

Reply:
Yup; I checked and you're correct - but it's not part of your changes, only one of warnings in legacy code.

Of course, fixing it would be appreciated (in PR dedicated only to this fix)… :)

I looked it up and unfortunately this change has potential for regression; in this case file access is redirected to PSP read, which might support up to 2^16 file descriptors (value is stored in max_files field in PSP); default limit is merely 20, but it can be raised up to 2^16 via interrupt 0x21 AH 0x67…

In other words: yes, the fix is needed in here, but very likely a different one, as to avoid introducing an edge-case bug.